### ISSUE TITLE
some weird graphical Issues

### ISSUE THREAD

**Issue by ProxyPlayerHD** [iChun/GeneralLaymansAestheticSpyingScreen#5]
versions:
Forge: 14.23.0.2552
GLASS: 1.12.2-7.0.0
Optifine: HD U C7 pre

A thing that happens when activating the Projector. Chunks not being loaded, maybe because i'm like 3 chunks away from the Camera thingy. And one block is not even changing at all.
![GIF](https://i.gyazo.com/c7ca071182552ec62330feaba5639317.gif)

and this is when i got Optifine too.
![GIF](https://i.gyazo.com/b93418cdbe8d4fb1cf6ac86bea63d13c.gif)

**Comment by iChun** [comment]
Yeah I have no support for Optifine or it's shaders. I've already made it
so that it doesn't crash which is good enough for me.

**Comment by ProxyPlayerHD** [comment]
well then rip GLASS. without Optifine my game runs like shit.

and what about that that first GIF tho? any idea what is causing that block and chunks?
